pages/common/opt.md

```diff
@@ -1,6 +1,6 @@
 # opt
 
-> A tool that takes LLVM source files and runs specified optimizations and/or analyses on them.
+> A tool that takes LLVM source files and runs specified optimizations and/or analysis on them.
```

Comment:
Nope, I'm not sure this is right. `analyses` is plural, but `analysis` is singular.

Reply:
Ahh, actually that one wasn't a find and replace.
I'm just an idiot and read it like "analyze on them" before which, which is wrong.

I'll fix that later!